### PR TITLE
feat(charts): Declarative chart component to render chart using plotly json schema

### PR DESCRIPTION
**Pull request by AtishayMsft**
## New Behavior

Created a declarative chart component to render our chart using a plotly JSON schema.
The chart types supported are:
- Line
- Area
- Vertical Bar
- Horizontal Bar
- Donut
- Pie
- Sankey
- Heatmap
- Gauge

## Related Issue(s)

- Fixes #

### CODE REVIEW

File: packages/charts/react-charting/etc/react-charting.api.md [hunk 1]
```diff
@@ -119,6 +119,14 @@ export const DataVizPalette: {
     highSuccess: string;
```
Comment (fabricteam) [inline packages/charts/react-charting/etc/react-charting.api.md:119]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33348&commitId=556bf0e69294b399cc1ae4e38dceb49f40ccb6be&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Bottom left edge.chromium.png|2309| Changed|
|Callout.Rendering callout attached to a rectangle.chromium.png|1493| Changed|
|Callout.Right bottom edge.chromium.png|3005| Changed|
|Callout.Bottom auto edge.chromium.png|2309| Changed|
|Callout.Top left edge.chromium.png|2307| Changed|
|Callout.Right top edge.chromium.png|1127| Changed|
|Callout.Bottom Left Edge RTL.chromium.png|2178| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUIV0 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>

File: apps/public-docsite-resources/config/api-docs.js [hunk 2]
```diff
@@ -108,6 +108,7 @@ module.exports = {
       'MultiStackedBarChart',
```
Comment (fabricteam) [inline apps/public-docsite-resources/config/api-docs.js:108]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33348&commitId=1c9ed8a74fd0503b44c135a6dccee686a23457bc&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|4| Changed|
|Avatar Converged.badgeMask.chromium.png|5| Changed|

</details>


  <details><summary><b>Drawer</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|2219| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|2818| Changed|
|Drawer.overlay drawer full.chromium.png|1139| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Comment (fabricteam) [inline apps/public-docsite-resources/config/api-docs.js:108]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentui-web-components-v3 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33348&commitId=86505a547befb783ae7b098d158edd969db0d4d1&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Image.chromium.png|248| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Comment by fabricteam** [comment]
## 🕵 fluentui-web-components-v3 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Review by krkshitij — APPROVED** [review]

**Pull request merged**